sagemaker_neo_compilation_jobs/xgboost_customer_churn/xgboost_customer_churn_neo.ipynb

```diff
@@ -87,7 +92,7 @@
     "from IPython.display import display\n",
     "from time import strftime, gmtime\n",
     "import sagemaker\n",
```

Comment:
We already imported sagemaker. Remove this call.

Reply:
Fixed